lib/plaindate.ts

```diff
@@ -216,7 +216,9 @@ export class PlainDate implements Temporal.PlainDate {
   toZonedDateTime(item: Params['toZonedDateTime'][0]): Return['toZonedDateTime'] {
     if (!ES.IsTemporalDate(this)) throw new TypeError('invalid receiver');
 
-    let timeZone, temporalTime;
+    let timeZone: Temporal.TimeZoneProtocol;
+    type TimeZoneAndPlainTimeProps = Exclude<typeof item, Temporal.TimeZoneProtocol | string>;
```

Comment:
to keep changes from temporal-polyfill minimized, maybe move this type declaration up one line, and then leave the single-line `let` statement to declare both variables?

Reply:
Done.